deployment.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deployment
+layout: recommendation
+expires: 2017-12-01
+---
+
+Teams should create standalone artifacts which can:
+
+- have automated tests run against them to ensure they work as expected
+- be deployed to a pre-production environment for manual testing
+- be promoted to a production environment
+
+The artifact should contain the running application, including
+any dependencies.
```

Comment:
Related to Richard's and David's comments: it would be good to describe *why* this is desired. Is it for:

- consistency: that one deployment behaves like another?
- security: that dependencies haven't been modified?
- availability: that you can deploy without depending on third-party services?
- or: some combination of the above?

For example, is using a dependency manager that pins the versions of dependencies and sub-dependencies sufficient?